tests/tracking/test_client.py

```diff
@@ -176,37 +176,31 @@ def test_client_registry_operations_raise_exception_with_unsupported_registry_st
             assert exc.value.error_code == ErrorCode.Name(FEATURE_DISABLED)
 
 
-def test_update_registered_model_compatibility_layer(mock_registry_store):
+def test_update_registered_model(mock_registry_store):
     """
-    This test makes sure that old (now deprecated) apis work as expected after api update.
-    Update registered model no longer accepts new name, but the client should translate it to
-    rename call.
+    Update registered model no longer supports name change.
```

Comment:
do we really need these deprecation tests - as something that will run forever? makes sense for this PR, but seems unnecessary for the future (we have other tests calling these methods, and if someone tries to use the old arguments, they will see python error thrown).

Reply:
I removed the deprecated part so now these test are only the legit non-deprecated arguments to `update_*` methods. Maybe these methods can move elsewhere.

Reply:
Ah, I see, these are the actual only tests for `update_registered_model` and `update_model_version`. I was confused because of the comment that suggested that we were specifically (and only) testing the deprecated behavior.